src/components/content.js

```diff
@@ -698,15 +699,27 @@ class Content extends React.Component {
     if (this.props.readOnly) return
     if (!this.isInEditor(event.target)) return
 
-    event.preventDefault()
-    const data = getTransferData(event.clipboardData)
+    const handleData = (data) => {
```

Comment:
Can you refactor this such that `handleData` is removed, by pulling these two lines out to above the `if`:

```js
data.isShift = !!this.tmp.isShifting
debug('onPaste', { event, data })
```

And by then just using `this.props.onPaste` directly in both places. Thanks!